drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -61,7 +62,12 @@ static struct nrf5_802154_data nrf5_data;
 #define ACK_REQUEST_BIT (1 << 5)
 #define FRAME_PENDING_BYTE 1
 #define FRAME_PENDING_BIT (1 << 4)
-#define TXTIME_OFFSET_US  (5 * USEC_PER_MSEC)
+#define TXTIME_OFFSET_US (1 * USEC_PER_MSEC)
+
+#define DRX_SLOT_PH 0
+#define DRX_SLOT_RX 1
+#define PH_DURATION 10
+#define XTAL_ACCURACY 20
```

Comment:
This value should be configurable. I think it should be taken from devicetree. Alternatively, kconfig could be used.